README.md

```diff
@@ -1,7 +1,7 @@
 ![UI5 icon](https://raw.githubusercontent.com/SAP/ui5-tooling/master/docs/images/UI5_logo_wide.png)
 
 # ui5-project
-> Modules for building a UI5 projects dependency tree, including configuration  
+> Modules for building a UI5 projects dependency tree, including configuration.
```

Comment:
It was actually intended to have a line-break here (two spaces at the end of the line).
Right now this sentence is in the same line as the next one ("Part of the...")

Reply:
Fixed